src/transformers/models/aria/configuration_aria.py

```diff
@@ -144,6 +144,11 @@ class AriaTextConfig(PretrainedConfig):
         "layers.*.mlp.up_proj": "colwise",
         "layers.*.mlp.down_proj": "rowwise",
     }
+    base_model_pp_plan = {
+        "embed_tokens": [["input_ids"], ["inputs_embeds"]],
+        "layers": [["hidden_states", "attention_mask"], ["hidden_states"]],
+        "norm": [["hidden_states"], ["hidden_states"]],
```

Comment:
since it's always (input,output) maybe a tuple here is better than a list, for fixed length